hedera-node/src/main/java/com/hedera/services/legacy/evm/SolidityExecutor.java

```diff
@@ -269,27 +269,27 @@ private void call() {
 				errorCode = CONTRACT_BYTECODE_EMPTY;
 				setError(String.format("Error: Bytecode is empty for contract 0x%s", CommonUtils.hex(targetAddress)));
 			} else {
-				var programInvoke = programInvokeFactory.createProgramInvoke(
-						solidityTxn, block, trackingRepository, repository, NULL_BLOCK_STORE);
-				((ProgramInvokeImpl) programInvoke).setStaticCall(localCall);
-				this.vm = new VM(config, VMHook.EMPTY);
-				this.program = new Program(
-						repository.getCodeHash(targetAddress),
-						code,
-						programInvoke,
-						solidityTxn,
-						config,
-						VMHook.EMPTY,
-						contractCreateAdaptor,
-						fundingAddress,
-						rbh,
-						sbh,
-						dynamicProperties.defaultContractLifetime(),
-						dynamicProperties.cacheRecordsTtl()).withCommonConfig(commonConfig);
+//				var programInvoke = programInvokeFactory.createProgramInvoke(
+//						solidityTxn, block, trackingRepository, repository, NULL_BLOCK_STORE);
+//				((ProgramInvokeImpl) programInvoke).setStaticCall(localCall);
+//				this.vm = new VM(config, VMHook.EMPTY);
+//				this.program = new Program(
+//						repository.getCodeHash(targetAddress),
+//						code,
+//						programInvoke,
+//						solidityTxn,
+//						config,
+//						VMHook.EMPTY,
+//						contractCreateAdaptor,
+//						fundingAddress,
+//						rbh,
+//						sbh,
+//						dynamicProperties.defaultContractLifetime(),
+//						dynamicProperties.cacheRecordsTtl()).withCommonConfig(commonConfig);
```

Comment:
Should we just remove all the commented out source lines in these files? If we need to see how it used to be done, we can look in git history.